calibre-web.json

```diff
@@ -0,0 +1,59 @@
+{
+  "Calibre Web": {
+    "description": "Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using an existing Calibre database.<p>The Web interface allows user logins and the management and download of e-Books through a clean web interface.</p><p>For more information please see <a href='https://github.com/janeczku/calibre-web' target='_blank' rel='noopener noreferrer'>the oficial Github page</a> and <a href='https://hub.docker.com/r/linuxserver/calibre-web' target='_blank' rel='noopener noreferrer'>the Docker container page.</a></p></br>If linking to the Calibre Rockon use the directory '/config/Calibre Library' as your database location during configuration in the WebUI.",
+    "more_info": "",
+    "version": "0.0.1",
+    "website": "https://github.com/janeczku/calibre-web",
+    "ui": {
+      "slug": ""
+    },
+    "volume_add_support": true,
+    "containers": {
+      "calibreWeb": {
+        "image": "linuxserver/calibre-web",
+        "launch_order": 1,
+        "opts": [
+          [
+            "-v",
+            "/config",
+            "-v",
+            "/books"
+          ]
+        ],
+        "ports": {
+          "8083": {
+            "description": "Calibre-web UI port.",
+            "host_default": 8083,
+            "label": "WebUI port",
+            "protocol": "tcp",
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/config": {
+            "description": "Choose the Share where you store Calibre configuration data. Eg: The Share called calibre-config.<p>If using the Calibre rockon, this must be the configuration share of Calibre.  When configuring Calibre-Web, use '/config/Calibre Library' as the database location.</p>",
```

Comment:
@Superfish1000 , can you comment on the need to select the same share as for the Calibre Rock-On? This usually tends to create a lot of issues with other docker images but it may be different here if the Calibre and Calibre-web Rock-Ons are designed to work together.
In my test, I selected:
- a new, dedicated share for calibre-web config
- the same MEDIA share for the `/books` directory so that Calibre-web could access Calibre books database.

It seemed to work in my very quick test, but I didn't go very far wit that test, so I might be missing an important point here.

Reply:
The Calibre-web server needs access to the Calibre database file.  This file is stored in the config directory of the Calibre RockOn in the case that they are being used together.  Calibre-web requires access to both of these shares as it is simply tapping into the data stored both in the Calibre database and the media directory.  The metadata for eBooks is stored in the Database and the books themselves are stored in the media share.